### PR TITLE
User roles micdra7

### PR DESCRIPTION
**Pull request by micdra7**

### CODE REVIEW

**Comment by Siemko** [comment]
Hi there @micdra7, please pull the newest master and resolve conflicts. We want to check your commit :)

**Comment by micdra7** [comment]
@Siemko got it done :)

File: server/src/modules/auth/guards/user-roles.guard.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { Reflector } from '@nestjs/core';
+
+@Injectable()
+export class RolesGuard implements CanActivate {
+  constructor(private readonly reflector: Reflector) {}
+
+  canActivate(
+    context: ExecutionContext,
+  ): boolean | Promise<boolean> | Observable<boolean> {
+    const roles = this.reflector.get<string[]>('roles', context.getClass());
```
[the remark this answers is not in the document]
Comment (Siemko) [reply server/src/modules/auth/guards/user-roles.guard.ts]: Furthermore, you could use Enum here, not string.

**Comment by micdra7** [comment]
@Siemko I've fixed all the issues you mentioned

**Pull request closed without merging**